### PR TITLE
[DOCS] FDS Deployment Pattern Redshift

### PR DESCRIPTION
**Pull request by kenwade4**
- [ ] Description of PR changes above includes a link to [an existing GitHub issue](https://github.com/great-expectations/great_expectations/issues)
- [ ] PR title is prefixed with one of: [BUGFIX], [FEATURE], [DOCS], [MAINTENANCE], [CONTRIB]
- [ ] Code is linted

    ```
    black .

    ruff . --fix
    ```
- [ ] Appropriate tests and docs have been updated

For more details, see our [Contribution Checklist](https://docs.greatexpectations.io/docs/contributing/contributing_checklist), [Coding style guide](https://docs.greatexpectations.io/docs/contributing/style_guides/code_style), and [Documentation style guide](https://docs.greatexpectations.io/docs/contributing/style_guides/docs_style).

After you submit your PR, keep the page open and **monitor the statuses of the various checks made by our continuous integration process at the bottom of the page. Please fix any issues that come up** and [reach out on Slack](https://greatexpectations.io/slack) if you need help. Thanks for contributing!

### CODE REVIEW

**Comment by ghost** [comment]
<details open>
<summary><b>👇 <a href="https://app.codesee.io/r/reviews?pr=7868&src=https%3A%2F%2Fgithub.com%2Fgreat-expectations%2Fgreat_expectations">Click on the image</a> for a new way to code review</b>
</summary>

####

[Review these changes using an interactive CodeSee Map](https://app.codesee.io/r/reviews?pr=7868&src=https%3A%2F%2Fgithub.com%2Fgreat-expectations%2Fgreat_expectations)

#### Legend
CodeSee Map legend
</details>

File: docs/docusaurus/docs/deployment_patterns/how_to_use_gx_with_aws/how_to_use_gx_with_aws_using_redshift.md [hunk 1]
```diff
@@ -68,18 +63,10 @@ import IdentifyDataContextExpectationsStore from '@site/docs/guides/setup/config
 
 import AddS3ExpectationsStoreConfiguration from '@site/docs/guides/setup/configuring_metadata_stores/components_how_to_configure_an_expectation_store_in_amazon_s3/_update_your_configuration_file_to_include_a_new_store_for_expectations_on_s.mdx'
 
-<!-- 1.4.3 Verify that the new Amazon S3 Expectations Store has been added successfully -->
-
-import VerifyS3ExpectationsStoreExists from '@site/docs/guides/setup/configuring_metadata_stores/components_how_to_configure_an_expectation_store_in_amazon_s3/_confirm_that_the_new_expectations_store_has_been_added_by_running_great_expectations_store_list.mdx'
```
Comment (donaldheppner) [inline docs/docusaurus/docs/deployment_patterns/how_to_use_gx_with_aws/how_to_use_gx_with_aws_using_redshift.md:73]: if these files aren't references elsewhere, can they be deleted?

Comment (kenwade4) [reply docs/docusaurus/docs/deployment_patterns/how_to_use_gx_with_aws/how_to_use_gx_with_aws_using_redshift.md:73]: I'll grep around

Comment (kenwade4) [reply docs/docusaurus/docs/deployment_patterns/how_to_use_gx_with_aws/how_to_use_gx_with_aws_using_redshift.md:73]: All of the various md and mdx files that are no longer referenced in this PR are being used by at least 1 other file.

File: docs/docusaurus/docs/guides/setup/configuring_metadata_stores/components_how_to_configure_an_expectation_store_in_amazon_s3/_identify_your_data_context_expectations_store.mdx [hunk 2]
```diff
@@ -1,6 +1,6 @@
 import TechnicalTag from '@site/docs/term_tags/_tag.mdx';
 
-You can add a new <TechnicalTag tag="expectation_store" text="Expectation Store" /> to your <TechnicalTag tag="data_context" text="Data Context" /> by using the `add_store()` metho by using the `add_store()` methodd.
+You can add a new <TechnicalTag tag="expectation_store" text="Expectation Store" /> to your <TechnicalTag tag="data_context" text="Data Context" /> by using the `add_store()` metho by using the `add_store()` method.
```
Comment (anthonyburdi) [inline docs/docusaurus/docs/guides/setup/configuring_metadata_stores/components_how_to_configure_an_expectation_store_in_amazon_s3/_identify_your_data_context_expectations_store.mdx]:
```suggestion
You can add a new <TechnicalTag tag="expectation_store" text="Expectation Store" /> to your <TechnicalTag tag="data_context" text="Data Context" /> by using the `add_store()` method.
```

Comment (anthonyburdi) [reply docs/docusaurus/docs/guides/setup/configuring_metadata_stores/components_how_to_configure_an_expectation_store_in_amazon_s3/_identify_your_data_context_expectations_store.mdx]: Also this line feels out of place, the section heading is `1.4.1 Identify your Data Context Expectations Store` not to add a store. Could this instead be consistent with the text in the validations store section: https://deploy-preview-7868--niobium-lead-7998.netlify.app/docs/deployment_patterns/how_to_use_gx_with_aws/how_to_use_gx_with_aws_using_redshift#151-identify-your-data-contexts-validation-results-store
```
You can find your [Validation Results Store](https://deploy-preview-7868--niobium-lead-7998.netlify.app/docs/terms/validation_result_store)A connector to store and retrieve information about objects generated when data is Validated against an Expectation Suite. configuration within your [Data Context](https://deploy-preview-7868--niobium-lead-7998.netlify.app/docs/terms/data_context).

Look for the following section in your [Data Context's](https://deploy-preview-7868--niobium-lead-7998.netlify.app/docs/terms/data_context) great_expectations.yml file:
```

**Review by anthonyburdi — CHANGES_REQUESTED** [review]
Just a small change, otherwise looks good!

File: docs/docusaurus/docs/guides/setup/configuring_metadata_stores/components_how_to_configure_an_expectation_store_in_amazon_s3/_identify_your_data_context_expectations_store.mdx (hunk 2 above)
Comment (anthonyburdi) [reply docs/docusaurus/docs/guides/setup/configuring_metadata_stores/components_how_to_configure_an_expectation_store_in_amazon_s3/_identify_your_data_context_expectations_store.mdx]: @kenwade4 I committed the suggestion above, but looking for the second item to be done before approving: https://github.com/great-expectations/great_expectations/pull/7868#discussion_r1192645463

**Review by anthonyburdi — APPROVED** [review]
LGTM!

**Pull request merged**